ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/DisplayGrid/DisplayGrid.cpp

```diff
@@ -144,7 +144,7 @@ void DisplayGrid::changeLabelColor(const QString& color)
       constexpr float size = 14.0f;
```

Comment:
move out of the loop?

Reply:
IMO bad idea, I prefer proper scoping over micro-optimisations the compiler should pick up on anyway.